### ISSUE TITLE
YOLOV8 Training Failure: Annotations Not Recognized When Exporting in YOLO Format

### ISSUE THREAD

**Issue by wsdltsai** [CVHub520/X-AnyLabeling#430]
Freezing layer 'model.22.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...
AMP: checks passed ✅
train: Scanning /workspace/datasets/mine_sc/train/labels... 238 images, 72 backgrounds, 0 corrupt: 100%|██████████| 310/310 [00:00<00:00, 946.22it/s]
train: New cache created: /workspace/datasets/mine_sc/train/labels.cache
val: Scanning /workspace/datasets/mine_sc/val/labels... 17 images, 27 backgrounds, 0 corrupt: 100%|██████████| 44/44 [00:00<00:00, 1465.21it/s]
val: New cache created: /workspace/datasets/mine_sc/val/labels.cache
Plotting labels to runs/detect/train29/labels.jpg...
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically...
optimizer: AdamW(lr=0.001, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0)
Image sizes 640 train, 640 val
Using 8 dataloader workers
Logging results to runs/detect/train29
Starting training for 1000 epochs...
导出yolo格式，标注小数位13位，无法识别训练

**Comment by CVHub520** [comment]
> Freezing layer 'model.22.dfl.conv.weight' AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n... AMP: checks passed ✅ train: Scanning /workspace/datasets/mine_sc/train/labels... 238 images, 72 backgrounds, 0 corrupt: 100%|██████████| 310/310 [00:00<00:00, 946.22it/s] train: New cache created: /workspace/datasets/mine_sc/train/labels.cache val: Scanning /workspace/datasets/mine_sc/val/labels... 17 images, 27 backgrounds, 0 corrupt: 100%|██████████| 44/44 [00:00<00:00, 1465.21it/s] val: New cache created: /workspace/datasets/mine_sc/val/labels.cache Plotting labels to runs/detect/train29/labels.jpg... optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... optimizer: AdamW(lr=0.001, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0) Image sizes 640 train, 640 val Using 8 dataloader workers Logging results to runs/detect/train29 Starting training for 1000 epochs... 导出yolo格式，标注小数位13位，无法识别训练

您好，这个应该跟标注小数位没有关系，可以仔细检查导出的标签，有可能是你导出的方式不对。另外可以贴一下具体的报错详情信息，如果方便的话可以提供几个图片和标签样本打包到邮箱：cv_hub@163.com

**Comment by wsdltsai** [comment]
你好
问题我昨天找到了，因为我用了不同的标注工具，您的这个的小数位是13位，labelimg是6位，混合在一起了，才出现那个问题。单独使用您的这个工具导出的标注可以训练。
非常感谢。
305874737
***@***.***
---- 回复的原邮件 ----
***@***.***>发送日期2024年5月23日 ***@***.***>***@***.***>,
***@***.***>主题Re: [CVHub520/X-AnyLabeling] 导出yolo格式，使用YOLOV8训练，标注无法识别 (Issue #430)
Freezing layer 'model.22.dfl.conv.weight' AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n... AMP: checks passed ✅ train: Scanning /workspace/datasets/mine_sc/train/labels... 238 images, 72 backgrounds, 0 corrupt: 100%|██████████| 310/310 [00:00<00:00, 946.22it/s] train: New cache created: /workspace/datasets/mine_sc/train/labels.cache val: Scanning /workspace/datasets/mine_sc/val/labels... 17 images, 27 backgrounds, 0 corrupt: 100%|██████████| 44/44 [00:00<00:00, 1465.21it/s] val: New cache created: /workspace/datasets/mine_sc/val/labels.cache Plotting labels to runs/detect/train29/labels.jpg... optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... optimizer: AdamW(lr=0.001, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0) Image sizes 640 train, 640 val Using 8 dataloader workers Logging results to runs/detect/train29 Starting training for 1000 epochs... 导出yolo格式，标注小数位13位，无法识别训练
***@***.***
